portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -71,108 +72,86 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         String comment = (String) entity.getProperty("text");
         String nickname = (String) entity.getProperty("nickname");
         String email = (String) userService.getCurrentUser().getEmail();
-        String image = (String) entity.getProperty("imgUrl");
-        ArrayList<String> messageReplies = (ArrayList) entity.getProperty("replies");
-        BlogMessage message = new BlogMessage(messageId, tag, comment, image, nickname, email, messageReplies, timestamp);
+        //String image = (String) entity.getProperty("imgUrl");
```

Comment:
Commented out code shouldn't be checked into production. The main reasons are that it makes things less readable and it is easily forgotten about.

To preserve code like this, you can start a branch that you intend to work on later, or you can copy and paste from github's history.